scene/gui/color_picker.h

```diff
@@ -68,6 +69,7 @@ class ColorPicker : public BoxContainer {
 	bool edit_alpha;
 	Size2i ms;
 	bool text_is_constructor;
+	int presetsPerRow;
```

Comment:
Identifiers use `snake_case` in Godot's code base (as seen in other member variables around this one).